internal/provider/provider_test.go

```diff
@@ -52,6 +53,33 @@ func TestWorkspace(t *testing.T) {
 	})
 }
 
+func TestHost(t *testing.T) {
+	resource.Test(t, resource.TestCase{
+		Providers: map[string]*schema.Provider{
+			"coder": provider.New(),
+		},
+		IsUnitTest: true,
+		Steps: []resource.TestStep{{
+			Config: `
+			provider "coder" {
+			}
+			data "coder_host" "me" {
+			}`,
+			Check: func(state *terraform.State) error {
+				require.Len(t, state.Modules, 1)
+				require.Len(t, state.Modules[0].Resources, 1)
```

Comment:
is this just ensuring properties exist? when would this fail?

Reply:
This just ensures it exists, otherwise it would panic below. Not entirely necessary for this test, just makes the output a bit nicer in case of a dev mistake!

Reply:
👍🏼